internal/clients/sources/stubs/sources_stub.go

```diff
@@ -21,7 +21,7 @@ type SourcesClientV2Stub struct{}
 
 func init() {
 	// We are currently using SourcesClientV2Stub
-	sources.GetSourcesClientV2 = getSourcesClientV2Stub
+	clients.GetSourcesClient = getSourcesClientV2Stub
```

Comment:
`getSourcesClientV2Stub` Maybe this should be changed as well?

Reply:
Done :)